internal/scheme/scheme.go

```diff
@@ -1,38 +1,22 @@
 package scheme
 
 import (
-	"github.com/kubewarden/audit-scanner/internal/constants"
 	policiesv1 "github.com/kubewarden/kubewarden-controller/pkg/apis/policies/v1"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
-	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/client-go/kubernetes/scheme"
 	wgpolicy "sigs.k8s.io/wg-policy-prototypes/policy-report/pkg/api/wgpolicyk8s.io/v1alpha2"
 )
 
-func NewScheme() *runtime.Scheme {
+func NewScheme() (*runtime.Scheme, error) {
 	scheme := scheme.Scheme
+	err := policiesv1.AddToScheme(scheme)
```

Comment:
I think this allows us to drop several unused constants from internal/constants/constants.go now.